libs/sparkpost-lib/src/test/java/com/sparkpost/EndPointTest.java

```diff
@@ -43,7 +43,7 @@ public void testSimpleEndPoint() {
         endPoint.addParam("num_rcpt_errors", 3);
 
         String result = endPoint.toString();
-        Assert.assertEquals("transmissions?num_rcpt_errors=3", result);
+        Assert.assertEquals("/transmissions?num_rcpt_errors=3", result);
```

Comment:
Great point. I was testing it as you can see but I think it would be a good test to verify that a leading slash is added specifically.

I will add this test.